apps/minifront/src/components/layout.tsx

```diff
@@ -3,21 +3,24 @@ import { HeadTag } from './metadata/head-tag';
 import { Header } from './header/header';
 import { Toaster } from '@repo/ui/components/ui/toaster';
 import { Footer } from './footer/footer';
-import { MotionConfig } from 'framer-motion';
 import '@repo/ui/styles/globals.css';
+import { SyncingModal } from './syncing-modal';
 
 export const Layout = () => {
   return (
-    <MotionConfig transition={{ duration: 0.1 }}>
```

Comment:
This is already included in `PenumbraUIProvider`, which is now at the top of the component tree in `main.tsx`